app/views/file_attachments/confirm_delete.html.erb

```diff
@@ -25,7 +29,8 @@
       <div class="govuk-body">
         <%= link_to "Cancel",
                   back_link_path,
-                  class: "govuk-link govuk-link--no-visited-state" %>
+                  class: "govuk-link govuk-link--no-visited-state", 
+                  data: { "modal-action": "cancel" } %>
```

Comment:
Minor: I think we expect 'cancel' to behave the same as 'back'. What do you think about re-using 'back' here?

Reply:
Minor: looking at the other four places we have a 'Cancel' action, we should probably use a slightly more specific name here e.g. `cancel-delete-attachment`.

Reply:
Using 'back' did occur to me as it is functionally identical but in the end I thought maybe it was good to be a bit more explicit in order to reinforce the idea that they are two seperate user actions. Having said that, I think i'm more inclined just to remove cancel entirely and use back. At least until we develop a policy regarding cancel.